testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -2735,35 +2731,51 @@ QUnit.module('positioning', {
             const visualPosition = this.getPosition();
             const expectedPosition = $target.position();
 
-            assert.deepEqual(visualPosition, expectedPosition, 'position is correct');
+            assert.deepEqual(visualPosition, expectedPosition);
         });
 
-        QUnit.test('popup should not restore position after fullScreen disable', function(assert) {
-            const visualPositionBeforeFullScreen = this.getPosition();
+        QUnit.test('popup should render on top-center position after change fullScreen option to false', function(assert) {
```

Comment:
Actually, not always at this position
```suggestion
        QUnit.test('popup should render on passed position after change fullScreen option to false', function(assert) {
```